frontend/src/scenes/max/Thread.tsx

```diff
@@ -261,23 +263,7 @@ function MessageGroup({ messages, isFinal: isFinalGroup }: MessageGroupProps): J
                         return <MultiVisualizationAnswer key={key} message={message} />
                     } else if (isReasoningMessage(message)) {
                         return (
-                            <MessageTemplate key={key} type="ai">
-                                <div className="flex items-center gap-2">
-                                    <img
-                                        src="https://res.cloudinary.com/dmukukwp6/image/upload/loading_bdba47912e.gif"
-                                        className="size-7 -m-1" // At the "native" size-6 (24px), the icons are a tad too small
-                                    />
-                                    <span className="font-medium">{message.content}…</span>
```

Comment:
Also, we lost the "…"
It made the process feel more open-ended to me, but no strong opinion